airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/ReplicationView.tsx

```diff
@@ -170,6 +178,16 @@ export const ReplicationView: React.FC<ReplicationViewProps> = ({ onAfterSaveSch
   return (
     <Content>
       <Card>
+        {schemaUpdateModalOpen &&
+          !diffAcknowledged &&
+          connection.catalogDiff?.transforms &&
+          connection.catalogDiff.transforms.length > 0 && (
+            <CatalogDiffModal
+              catalogDiff={connection.catalogDiff}
+              catalog={connection.syncCatalog}
+              setDiffAcknowledged={setDiffAcknowledged}
+            />
+          )}
```

Comment:
If we'll wait for https://github.com/airbytehq/airbyte/pull/14634 to be merged, we can use the ModalService introduced in this, to show this modal more via the new imperative API once the Source Scheme refresh call is done, which could improve this (since in general no state management needed anymore for it).

Reply:
ModalService sounds great.